PolyEngine/Engine/Src/Engine.cpp

```diff
@@ -106,3 +106,10 @@ void Engine::ResizeScreen(const ScreenSize & size)
 	gConsole.LogDebug("Screen resize: {} {}", size.Width, size.Height);
 	GetRenderingDevice()->Resize(size);
 }
+
+void Engine::QuitGame()
+{
+	shouldQuit = 1;
```

Comment:
Please nevere use numbers for boolean value assignments. Use `true` and `false` instead. Also this implementation can be written in one line.